src/Bicep.Core/Diagnostics/SimpleDiagnosticWriter.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.Core.Diagnostics
+{
+    public class SimpleDiagnosticWriter : IDiagnosticWriter
```

Comment:
Mind adding a comment to document the behavior of the diagnostic writer (why it doesn't record the diagnostics)?

Reply:
Good idea!